README.md

```diff
@@ -1,31 +1,31 @@
-##What Is the Chart Authoring Tool?##
+## What Is the Chart Authoring Tool?##
 
 A tool for authoring accessible charts for the web. Including a sonified (audio) version of the chart.
 
-##How Do I Get the Chart Authoring Tool?##
+## How Do I Get the Chart Authoring Tool?##
 
 You can checkout and fork the Chart Authoring Tool on github:
 
 <https://github.com/fluid-project/chartAuthoring>
 
-##Who Makes the Chart Authoring Tool, and How Can I Help?##
+## Who Makes the Chart Authoring Tool, and How Can I Help?##
 
 The Fluid community is an international group of designers, developers, and testers who focus on a common mission: improving the user experience and accessibility of the open web.
 
 The best way to join the Fluid Community is to jump into any of our community activities. Visit our [website](http://fluidproject.org/) for links to our mailing lists, chat room, wiki, etc.
 
-##Building
+## Building the Demos and Running the Tests
 
 Requirements:
 * `npm`
 
 Steps:
 * `npm install`
-    * postinstall should run the `grunt installFrontEnd` task after installation completes
+* Run the `grunt installFrontEnd` task after installation completes
```

Comment:
Probably add a little "note" paragraph to explain why running `grunt installFrontEnd` is no longer a npm postinstall step so it won't be questioned next time. Thanks.